subsystems/motor/speedmodifiers/CapSpeedModifier.java

```diff
@@ -0,0 +1,21 @@
+package org.usfirst.frc4904.standard.subsystems.motor.speedmodifiers;
+
+
+import org.usfirst.frc4904.standard.Util;
+
+public class CapSpeedModifier implements SpeedModifier {
+	public Util.Range speedCap;
```

Comment:
Also please make the speedCap either protected or private. If it needs to be accessed, add a getter